docs/DeveloperGuide.md

```diff
@@ -1,3 +1,1140 @@
+
+-   The input of the queried doctor ID is passed along as a string into
+    > the stated method and it iterates through the ArrayList of saved
+    > doctor's appointment with a for loop.
+
+-   Comparing if the Doctor ID while iterating is equals to the input
+    > that is passed, the details of the Appointment - ID, Patient's
+    > Name, Gender, Date, is printed out for the user.
+
+3.  delete \[Appointment ID\]:
+
+-   When the command delete is called. The Parser.java will recognise
+    > the command and call upon deleteAppointment() method in the
+    > AppointmentActions.java class.
+
+-   A variable "index" of default value 999 is initialised. A for loop
+    > is then called upon to iterate through the current ArrayList of
+    > doctor's appointment list to search for the ID that is equal to
+    > the input.
+
+-   If there is a match, the variable index would be set to the
+    > iteration in which the ID had been matched
+
+-   If the value of index remains at 999, the system recognise that
+    > there is no matched ID in the system, else, the system would
+    > remove the appointment details from the Doctor Appointment List
+    > and call upon writeToFile() method from the storage class to
+    > rewrite the file with the most current details.
+
+4.  help:
+
+-   When the command help is called. The Parser.java will recognise the
+    > command and call upon helpAppointment() method in the
+    > AppointmentActions.java class.
+
+-   The method calls upon doctorAppointmentHelp() method from the UI
+    > class which will print the commands that are necessary for the
+    > program.
+
+5.  return:
+
+-   When the command return is called, the Parser.java class returns a
+    > boolean value of True instead of the default False.
+
+-   This will set the boolean variable, isReturnToStartMenu in the
+    > DoctorAppointmentInstance.java class to True.
+
+-   This will prompt the program to return to the staff Menu as if when
+    > the program first booted.
+
+ 
+
+**Drugs-related features**
+--------------------------
+
+This feature can be accessed when the user inputs "5" into the terminal.
+The user will gain access to DrugParser, DrugStorage, and DrugActions.
+If data is found on the Drugs text file, it will be loaded into the
+arraylist, else a new text file is created. The purpose of this feature
+is to save a database of all the drugs in the inventory.
+
+[Implementation:]{.ul}
+
+User executes a command. Depending on the command, DrugInstance will
+call the relevant methods in DrugActions. DrugActions will either add,
+list or delete a Drug object.
+
+add\[Drug name\] \[Price\] \[Quantity\]:
+
+Getting User Input
+
+> User input add command which is processed by DrugInstance.run().
+
+Creating Drug object with User Input
+
+> DrugAction.addDrug() creates a new NurseSchedule object and is stored
+> into an existing ArrayList\<Drug\> inventory which contains all the inventory
+> objects.
+
```

Comment:
![image](https://user-images.githubusercontent.com/60333700/113230615-f74d8080-92cb-11eb-88eb-0cd4d86695cb.png)
The information under this section could have been presented in a better format. Perhaps of using ">" to add details of the features, I think you can use bulleted points or numbers to break these information into different steps.